osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,33 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
+                case "msg":
+                case "query":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage($"Usage: /{command} [user]"));
+                        break;
+                    }
+
+                    // Check if the user has joined requested channel already.
+                    var privateChannel = JoinedChannels.FirstOrDefault(
+                        c => c.Type == ChannelType.PM && c.Users.Count == 1 && c.Name.Equals(content, StringComparison.OrdinalIgnoreCase));
```

Comment:
Am I the only one weirded out by the fact that this is apparently checking the channel name rather than the actual user's username to check if the opened channel is already opened? It feels like a rather incidental usage.

It may be so that the two are equivalent, but I'm not sure whether that will always hold in the future...?

Reply:
After a restart the users in `c.Users` only have their ID set, so username is null. I don't really know where these channel objects are stored/loaded from, or if it is worth storing usernames with these channels if this is the only usage.

Reply:
I don't think there's need for that currently, you can only leave that as an inline comment above the code stating that it is not possible to check the PM user's username instead.

Reply:
Sure. There are tests that cover this now anyway.